frontend/packages/ceph-storage-plugin/src/utils/common.tsx

```diff
@@ -14,3 +19,14 @@ export const commaSeparatedString = (text: string[], t: TFunction): string =>
   );
 
 export const toList = (text: string[]): React.ReactNode => text.map((s) => <li key={s}>{s}</li>);
+
+export const calcPercentage = (value: number, total: number) =>
+  humanizePercentage((value * 100) / total).string;
+
+export const twelveHoursdateTimeNoYear = new Intl.DateTimeFormat(getLocale() || undefined, {
+  month: 'short',
+  day: 'numeric',
+  hour: 'numeric',
+  minute: 'numeric',
+  hour12: true,
+});
```

Comment:
Can you explain , why do we need this ?

Reply:
UX team wants 12 hours local time for mirroring status last checked (https://marvelapp.com/prototype/64gfhgf/screen/80676224), Datetime module doesn't have any functions to display that. So i created our own function.

Reply:
Basically,  hadas dont wonts to use UTC here,